control_msgs/msg/SingleInterfaceValue.msg

```diff
@@ -0,0 +1,6 @@
+std_msgs/Header header
+
+# Full qualified interface name e.g. "joint1/position"
+string interface_name
+# Value corresponding to the interface 
+float64 value
```

Comment:
```suggestion
# Full qualified interface name e.g. "joint1/position"
string interface_name
# Value corresponding to the interface 
float64 value
```
I would suggest moving this header to the DynamicInterfaceValues message as it would make more sense over there. As it will be a duplication when we have a vector.

Reply:
if you like my second suggestion, then we can remove this SingleIntyerfaceValue message